cmd/project/create_template.go

```diff
@@ -82,12 +82,12 @@ func getSelectionOptionsForCategory(clients *shared.ClientFactory) []promptObjec
 			Repository: "slack-cli#getting-started",
 		},
 		{
-			Title:      fmt.Sprintf("Automation app %s", style.Secondary("Custom steps and workflows")),
-			Repository: "slack-cli#automation-apps",
+			Title:      fmt.Sprintf("Agentic AI app %s %s", style.Secondary("Slack agents and assistants"), style.Emoji("sparkles")),
```

Comment:
```suggestion
			Title:      fmt.Sprintf("Agentic AI app %s", style.Secondary("Slack agents and assistants")),
```

suggestion(non-blocking): This highlights interesting features but I wonder if moving this to be the second option suffices? The `emoji` logic might otherwise output strange characters for Windows machines at this time...